content/articles/understanding-how-cluster-marker-works/index.md

```diff
@@ -0,0 +1,210 @@
+### Web GIS
+ Understanding how cluster markers work in web mapping using leaflet.js, as we are all aware that Big Data is the keyword for the twenty-first century. It's shocking to know that only a few people understand that a large part of data is categorized as geospatial. Nowadays nearly all of the gadgets we used are satellite-based systems like GPS. Today, the simplest case of displaying data/marker on a map with an acceptable user experience may be a real challenge. Especially for the user, they might find it difficult to understand to analyze data at a particular zoom level. For instance, a platform that has Users with hundreds of markers on the map. When you zoom out the map, these markers all overlap and make the map appears messy and crowded. To improve this kind of map, it is professional to use the marker clustering method, a technique of grouping markers that are close to each other together on each zoom level. 
```

Comment:
There appears to be some grammar issues in this paragraph:

`Understanding how cluster markers work in web mapping using leaflet.js, as we are all aware that Big Data is the keyword for the twenty-first century` - `these sentences do not go along`. It's shocking to know that only a few people understand that a large part of data is categorized as geospatial. Nowadays nearly all of the gadgets we `used` are satellite-based systems like GPS. Today, the simplest case of displaying data/marker on a map with an acceptable user experience may be a real challenge. `Especially for the user` - `it shouldn't start a sentence`, they might find it difficult to understand to analyze data at a particular zoom level. For instance, a platform that has `U`sers with hundreds of markers on the map. When you zoom out the map, these markers all overlap and make the map `appears` - appear messy and crowded. To improve this kind of `map` - `replace with synonym`, it is professional to use the marker clustering method, a technique of grouping markers that are close to `each other together on each zoom level` - eliminate repition `each`.